kolibri/plugins/coach/assets/src/views/common/StatusElapsedTime.vue

```diff
@@ -0,0 +1,219 @@
+        context:
+          'Indicates that an item was created a number of minutes prior to the current time, but the time is always less than 1 hour ago.',
+      },
+      createdHoursAgo: {
+        message: 'Created {hours} {hours, plural, one {hour} other {hours}} ago',
+        context:
+          'Indicates that an item was created a number of hours prior to the current time, but the time is always less than one day ago',
+      },
+      createdDaysAgo: {
+        message: 'Created {days} {days, plural, one {day} other {days}} ago',
+        context: 'Indicates that an item was created a number of days prior to the current date.',
+      },
+      closedSecondsAgo: {
+        message: 'Closed {seconds} {seconds, plural, one {second} other {seconds}} ago',
+        context:
+          'Indicates that an item was closed a number of seconds prior to the current time, but is always less than 1 minute ago.',
+      },
+      closedMinutesAgo: {
+        message: 'Closed {minutes} {minutes, plural, one {minute} other {minutes}} ago',
+        context:
+          'Indicates that an item was closed a number of minutes prior to the current time, but the time is always less than 1 hour ago.',
+      },
+      closedHoursAgo: {
+        message: 'Closed {hours} {hours, plural, one {hour} other {hours}} ago',
+        context:
+          'Indicates that an item was closed a number of hours prior to the current time, but the time is always less than one day ago',
+      },
+      closedDaysAgo: {
+        message: 'Closed {days} {days, plural, one {day} other {days}} ago',
+        context: 'Indicates that an item was closed a number of days prior to the current date.',
+      },
+      openedSecondsAgo: {
+        message: 'Opened {seconds} {seconds, plural, one {second} other {seconds}} ago',
+        context:
+          'Indicates that an item was opened a number of seconds prior to the current time, but is always less than 1 minute ago.',
+      },
+      openedMinutesAgo: {
+        message: 'Opened {minutes} {minutes, plural, one {minute} other {minutes}} ago',
+        context:
+          'Indicates that an item was opened a number of minutes prior to the current time, but the time is always less than 1 hour ago.',
+      },
+      openedHoursAgo: {
+        message: 'Opened {hours} {hours, plural, one {hour} other {hours}} ago',
+        context:
+          'Indicates that an item was opened a number of hours prior to the current time, but the time is always less than one day ago',
+      },
+      openedDaysAgo: {
+        message: 'Opened {days} {days, plural, one {day} other {days}} ago',
+        context: 'Indicates that an item was opened a number of days prior to the current date.',
+      },
+      secondsAgo: {
+        message: '{seconds} {seconds, plural, one {second} other {seconds}} ago',
+        context: 'Indicates that something occured seconds prior to the current time.',
```

Comment:
Missing one `r` in `occurred` (with 3 more instances below).

Reply:
Thanks! Fixed.